services/common/src/interfaces/index.ts

```diff
@@ -16,3 +12,7 @@ export * from "./mineType.interface";
 export * from "./mineTypeDetail.interface";
 export * from "./mineVerifiedStatus.interface";
 export * from "./mineWorkInformation.interface";
+export * from "./party";
+export * from "./tailingsStorageFacility";
```

Comment:
Looks like this same line of code also exists in line 4.

Reply:
Oh, good catch!  Yes, there is an index file in each subfolder now.